testing/btest/Baseline/spicy.file-analysis-data-in/output

```diff
@@ -1,3 +1,5 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
 FM47gX3vI5ofQPm1li
 FZjUS57tUkGFTibv3
+FaAaAaAaAaAaAaAaAa
```

Comment:
Nit: leakage between test cases. Making this two cases should fix that.

Reply:
Ok, that I kind of agree with. I admittedly was lazy here :)